commands/banAlts.js

```diff
@@ -0,0 +1,40 @@
+const Discord = require('discord.js');
+const mongo = require('../mongo.js');
+// const { assignRole } = require('../server/server');
+
+const settings = require('../settings.js');
+
+module.exports = {
+    name: 'banalts',
+    description: 'Bans all alts of a verified Discord user using their Waterloo ID.',
+    args: true,
+    guildOnly: true,
+    displayHelp: true,
+    permissions: ['MANAGE_GUILD'],
+    async execute(message, args) {
+        const userID = args
+        const guildSettings = settings.get(message.guild?.id);
+        if (!guildSettings.verificationEnabled) {
+            return message.reply('This server does not have verification enabled');
+        }
+
+        message.channel.send(`Attempting to ban all alternate accounts of the user ${userID}`);
+        await message.guild.members.fetch();
+
+        // get specified user's UWID
+        const { uwid: userUWID } = await mongo.getDB().collection("users").findOne({ discordId: userID.id })
+        const altCounter = 0
+        // find all other discord accounts that share this UWID
+        for await (const user of mongo.getDB().collection("users").find({ uwid: userUWID })) {
+          // probably not very efficient for large servers, could just try/catch
+          // errors if user doesn't exist instead
+          if (message.guild.members.cache.values().includes(user.discordId)) {
```

Comment:
Is this the right syntax? I believe the cache is a collection of <Snowflake, GuildMember>, and calling `.values()` would get `GuildMember`s. IIRC you can use `.has()` to fetch the guild member assuming the Snowflake (discordId) exists